src/main/java/gov/cabinetofice/gapuserservice/model/Blacklist.java

```diff
@@ -0,0 +1,24 @@
+package gov.cabinetofice.gapuserservice.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.*;
+import lombok.*;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.util.Date;
+
+@EntityListeners(AuditingEntityListener.class)
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table
+public class Blacklist {
+    @Id
+    private String jwt;
```

Comment:
Can we add an integer column to this table and use that as the ID please? JWTs can't be relied on to be unique